Sources/Workspace/InitPackage.swift

```diff
@@ -269,6 +281,22 @@ public final class InitPackage {
                                 ]),
                         ]
                     """
+                } else if packageType == .buildToolPlugin {
+                    param += """
+                            .plugin(
+                                 name: "\(typeName)",
+                                 capability: .buildTool()
+                             ),
+                        ]
+                    """
+                } else if packageType == .commandPlugin {
+                    param += """
+                            .plugin(
+                                 name: "\(typeName)",
+                                 capability: .command(intent: .custom(verb: "\(typeName)", description: "prints hello world"))
```

Comment:
nit: `description` on a new line

Reply:
SwiftFormat probably won't (and shouldn't!) format the contents of a multiline string but yes, I'll do that too.

Reply:
`swiftformat` with default options on the generated templates actually leaves this on the same line, but I'll move it